### DISCUSSION TITLE
Convert satpoint to sat #?

### DISCUSSION THREAD

**Discussion by domhofmann** [ordinals/ord#1629] [Q&A] [answered]
Is there any way to convert a satpoint in the UTXO list (specifically the satpoint that is chosen for the next inscription) to a sat other than creating an index with `--index-sats`? 

Assuming not given my understanding, but I'm having some trouble with a fully sat-indexed index (seems like it'll take nearly a week to complete on decent hardware). Separately, have there been any performance concerns or improvements for full indexing between the 0.4.2 tag and the current code in `master`?

**Comment by raphjaph** [comment, marked as answer]
Sats are assigned their ordinal number when they are mined so you have to build the index with `--index-sats` to use sats. 

Yes, the full index is very memory intensive to build. You need at least 64gb to do that. We're working on some improvements but they are not in `master` yet.

**Comment by domhofmann** [comment]
Thanks! Didn't realize memory was a constraint there, super helpful.
